Source/WebCore/rendering/svg/SVGRenderTreeAsText.cpp

```diff
@@ -445,7 +445,7 @@ void writeSVGResourceContainer(TextStream& ts, const LegacyRenderSVGResourceCont
     writeNameAndQuotedValue(ts, "id"_s, id);
 
     if (resource.resourceType() == MaskerResourceType) {
-        const auto& masker = static_cast<const LegacyRenderSVGResourceMasker&>(resource);
+        const auto& masker = downcast<const LegacyRenderSVGResourceMasker>(resource);
```

Comment:
Can we just use `dynamicDowncast` in the if condition?

Reply:
I think using `dynamicDowncast` as @szewai would be clearer:
```
if (auto* masker = dynamicDowncast<const LegacyRenderSVGResourceMasker>(resource)) {
```